app/src/main/res/layout/activity_multimeter.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+        <TableLayout
+            android:id="@+id/tablelayout_control_read1"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <!-- Resistance -->
+            <TableRow
+                android:id="@+id/tablerow_control_read1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/control_read_border_hdpi"
+                android:orientation="horizontal">
+
+                <Button
+                    android:theme="@style/AppTheme.Button"
+                    android:id="@+id/button_control_read1"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/resistance"
+                    />
+
+                <TextView
+                    android:id="@+id/tv_control_read1"
+                    android:layout_width="0dp"
+                    android:layout_height="40dp"
```

Comment:
You can add these files to the dimensions file and call it from there. You have done it correctly at some places.